internal/gengapic/testdata/foo_rest_client_init.want

```diff
@@ -0,0 +1,68 @@
+// internalFooClient is an interface that defines the methods availaible from Awesome Foo API.
+type internalFooClient interface {
+	Close() error
+	setGoogleClientInfo(...string)
+	Connection() *grpc.ClientConn
```

Comment:
Doesn't look like `Connection` is implemented. Probably OK to leave as a TODO.

Reply:
It's not currently possible to instantiate a rest client, even within this PR, so any breaking changes are at this point theoretical.